docs/specs/template.yml

```diff
@@ -15,6 +15,22 @@ repos:
 outputs:
   docs/a.json: |
     { "conceptual": "<div class=\"NOTE\"><p>my note</p><p>note</p></div>" }
+  .publish.json: |
+    {"theme_branch": "main"}
+---
```

Comment:
shall we add a case of none-master/main branch?

Reply:
updated